samples/TypeScript/js-require.tsx

```diff
@@ -0,0 +1 @@
+const React = require("react")
```

Comment:
Do you have samples from "real applications" we could use here?
These sample files are used to train the Bayesian classifier, we prefer to avoid using small/unrepresentative files.